src/modules/tests/tests.reducer.ts

```diff
@@ -116,8 +117,10 @@ const createStateObject = (state: TestsModel, action: Action, slotId: string) =>
             testSummary: testSummaryReducer,
             communicationPreferences: communicationPreferencesReducer,
           }, combineReducers,
-        // @ts-ignore
-        )(state.startedTests[slotId], action),
+        )(
+          state.startedTests[slotId] as Required<StandardCarTestCATBSchema>,
```

Comment:
Can we add an explanatory comment here for the benefit of posterity?
Something along the lines of -
`The redux pattern necessitates that the state tree be initialised with all its properties declared. This conflicts with the 'StandardCarTestCATBSchema' TS interface as many of its properties are optional (?). In order to reconcile the TS interface and the redux reducer pattern we use the TS 'Required' mapped type which The 'Required' type which strips ? modifiers from all properties of 'StandardCarTestCATBSchema', thus making all properties required.`